src/Button.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function Button({ children, onClick }) {
+  return <button type="button" onClick={onClick}>{children}</button>;
```

Comment:
```suggestion
  return (
    <button type="button" onClick={onClick}>
      {children}
    </button>
  );
```

가독성 측면에서 줄바꿈 처리를 해주는건 어떨까요?

Reply:
수정했습니다!